.gitignore

```diff
@@ -11,6 +11,14 @@ core/build-version.json
 npm-debug.log
 metadata.db
 welder-web.spec
-/package-lock.json
+cockpit-composer.spec
+package-lock.json
 /po/etag-cache.json
 /po/*.pot
+bots
+images
```

Comment:
I think it's safer to turn this in to `/test/images`. It's totally plausible that there is some othe `images` directory somewhere else, which contains a bunch of PNGs or so. Let's not ignore these :)